storm-multilang/dotnet/src/main/dotnet/Dotnet.Storm.Adapter/Components/Component.cs

```diff
@@ -93,9 +93,9 @@ public static VerificationResult VerifyOutput(string stream, List<object> tuple)
 
         protected string[] Arguments { get; private set; }
 
-        protected static IDictionary<string, object> Configuration { get; private set; }
+        public static IDictionary<string, object> Configuration { get; set; }
```

Comment:
Do we want to allow a user to set this values up?

Reply:
We are doing this for tests. For topology only, no we do not.